src/adtechmedia-config.php

```diff
@@ -31,7 +31,7 @@ class Adtechmedia_Config {
 		'tpl_js_url'                     => 'https://www.adtechmedia.io/atm-core/atm-build/atmTpl.js',
 		'tpl_mgmt_js_url'                => 'https://www.adtechmedia.io/atm-core/atm-build/atmTplManager.js',
 		'terms_url'                      => 'https://www.adtechmedia.io/terms/dialog.html',
-		'register_url_tpl'               => 'https://www.adtechmedia.io/admin/accounts/signup/%s',
+		'register_url_tpl'               => 'https://www.manage.adtechmedia.io/accounts/signup/%s',
```

Comment:
second level domain should look like manage.adtechmedia.io not www.manage.adtechmedia.io